cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -711,4 +726,16 @@ public static ClientBuilder bluemix(String vcapServices, String instanceName) {
                 String.format("Cloudant service instance matching name %s was not found.", instanceName)
         );
     }
+
+    public ClientBuilder iamApiKey(String iamApiKey) {
+        this.iamApiKey = iamApiKey;
+        return this;
+    }
+
+    public ClientBuilder iamServer(URL iamServer) {
```

Comment:
See c07ebeee7aab28094fa751034627ddb42a3b0ee7 - the exact property name is up for grabs